src/Arcus.EventGrid/Parsers/EventConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Arcus.EventGrid.Contracts;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Arcus.EventGrid.Parsers
+{
+    /// <summary>
+    /// Converter to control how the <see cref="Event"/> model gets serialized and deserialized.
+    /// </summary>
+    public class EventConverter : JsonConverter<Event>
```

Comment:
Why do we need this? 🤔

Reply:
This way, we can just call `JsonConvert.DeserializeObject<Event>`.

Reply:
Ok, cool!